web/client-config.js

```
@@ -1,6 +1,6 @@
 module.exports = {
   sanity: {
-    projectId: process.env.GATSBY_SANITY_PROJECT_ID || 'uin6lq3v',
+    projectId: process.env.GATSBY_SANITY_PROJECT_ID || 'on977l64',
```

Perhaps we should raise a hard error if either the project id or dataset is missing. I think I would rather the system stop than pick the wrong default.

General question about Sanity: Does it default to `production` for the `DATASET`? We prefer to default to development when/where possible since it's a safer default. It means we don't inadvertently change production data, for example.

---

Form what I've seen all their starters generally have production as the default. I thought that was weird too. I will make a development dataset to use and then pop that into the `.env.development` env vars.

---

I made a development dataset, but then I realized that the studio isn't grabbing the dataset and projectId from the env vars like on the web side. Maybe this is something we talk through to see the best way of setting this up.

---

Perfect 😗 👌